src/promptflow/promptflow/_sdk/operations/_flow_operations.py

```diff
@@ -377,7 +377,7 @@ def _build_as_executable(
             import streamlit_quill  # noqa: F401
         except ImportError as ex:
             raise UserErrorException(
-                f"Please install PyInstaller, streamlit and streamlit_quill for building " f"executable, {ex.msg}. "
+                f"Please install PyInstaller, bs4, streamlit and streamlit_quill for building executable, {ex.msg}. "
```

Comment:
I think we should not print the package name in the error message, as long as user install promptflow[executable]

Reply:
updated